minecraft_servers/anturnia/manifest.json

```diff
@@ -11,6 +11,8 @@
     "en"
   ],
   "social": {
+    "web": "https://anturnia.net",
+    "web_support": "https://discord.gg/gYsf49MzwH",
```

Comment:
We only support web pages like FAQs or contact forms. Use "discord" for a link to your discord.